### DISCUSSION TITLE
[$500 reward] Remove "using Snapchat for Web" status message when starting a conversation

### DISCUSSION THREAD

**Discussion by Ludotrico** [dclstn/better-chat#437] [Ideas]
My friends and I are willing to donate $500 to anyone that is able to figure out how to remove the "using Snapchat Web" status message when starting a conversation. 

![Screenshot 2024-05-21 at 12 05 33 AM](https://github.com/dclstn/better-snapchat/assets/44794706/bf2e7bf7-21bf-4a45-8e12-a948e2a0383b)

![5af86b6d-8f9c-439a-b702-2a3f199503b3](https://github.com/dclstn/better-snapchat/assets/44794706/a7733c01-ba88-4a4e-8e05-c7c459ef5030)

**Comment by Ludotrico** [comment]
Any luck on this? : ) @dclstn

**Comment by Ludotrico** [comment]
if we increase the reward, would that help? @dclstn

**Comment by dclstn** [comment]
Hi @Ludotrico the issue isn't money, it's just finding time to work on this. Reverse-engineering tends to be very time-inducing and convoluted, but I will get back to you sometime this month.

**Comment by arthurauffray** [comment]
I recommend using the snapchat mobile app, my Paypal is arthurauffray@gmail.com. Thanks

**Comment by Ludotrico** [comment]
> Hi @Ludotrico the issue isn't money, it's just finding time to work on this. Reverse-engineering tends to be very time-inducing and convoluted, but I will get back to you sometime this month.

@dclstn my friends and I have increased the reward to $500, any progress? : ) it's really important to us

**Comment by nexuxd** [comment]
did you ever find anything? need this aswell

**Comment by darkacorn** [comment]
even if you put the reward to 100 000 - this isnt from the plugin - its very much the way snapchat web announces it self .. they are different api's

**Comment by residex** [comment]
I was looking into this a while ago, it is definitely possible im happy to look into it for you if you still need it :)

telegram - @residex
